src/GovUk.Education.ExploreEducationStatistics.Admin/Requests/ReleaseSeriesItemRequests.cs

```diff
@@ -1,21 +1,22 @@
 #nullable enable
-
 using System;
+using System.ComponentModel.DataAnnotations;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Requests;
 
-public class ReleaseSeriesLegacyLinkAddRequest
+public record ReleaseSeriesLegacyLinkAddRequest
 {
-    public string Description { get; set; } = string.Empty;
-    public string Url { get; set; } = string.Empty;
+    [Required]
+    public required string Description { get; init; }
```

Comment:
They have different effects, yes. 

The `required` keyword was new in C#11 and means the field must be initialized at the same time the object is constructed and is enforced at compile time. For example, if you look in the changes to the 

we can no longer do this

```csharp
new ReleaseSeriesLegacyLinkAddRequest()
````

which we wouldn't really want to because it doesn't make sense to have a request with no values, and instead must do something like this

```csharp
new ReleaseSeriesLegacyLinkAddRequest
{
    Description = "Test description",
    Url = "https://test.url"
};
```

The `RequiredAttribute` used with `[Required] provides validation that the field has a value in the request.

I imagine that if you don't use the `[Required]` annotation but still use `required`, and the value is not present in the request, that the request won't fail validation and then there might be an error attempting to deserialise the JSON into the object due to the field marked as `required` not being initialised. I'd have to try this out to see if this is true. I think you are right to question it though because if this is true you might expect that type of error to be returned as a validation error, but I don't think that's currently how it works.